scripts/nyc/transform-coverage.ts

```diff
@@ -0,0 +1,64 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+interface Coverage {
+  [filePath: string]: FileCoverage
+}
+
+interface FileCoverage {
+  path: string;
+  inputSourceMap?: InputSourceMap;
+}
+
+interface InputSourceMap {
+  file: string;
+  sources: string[];
+}
+
+export function transformCoverageFiles(
+	projectRoot: string,
+  pathTransformer: (path: string) => string
+): void {
+  const nycOutput = path.join(projectRoot, '.nyc_output');
+
+  if (!fs.existsSync(nycOutput)) {
+    console.warn('No .nyc_output present...');
+    process.exit(0);
+  }
```

Comment:
```suggestion
  if (!fs.existsSync(nycOutput)) {
    console.warn('No .nyc_output present...');
    return;
  }
```

Reply:
Ah right, forgot to remove that after refactoring 🙈